tools/wptrunner/wptrunner/tests/test_formatters.py

```diff
@@ -0,0 +1,61 @@
+import json
+import sys
+import time
+from os.path import dirname, join
+from StringIO import StringIO
+
+from mozlog import handlers, structuredlog
+
+sys.path.insert(0, join(dirname(__file__), "..", ".."))
+
+from wptrunner.formatters import WptreportFormatter
+
+
+def test_wptreport_runtime(capfd):
+    # setup the logger
+    output = StringIO()
+    logger = structuredlog.StructuredLogger("test_a")
+    logger.add_handler(handlers.StreamHandler(output, WptreportFormatter()))
+
+    # output a bunch of stuff
+    logger.suite_start(["test-id-1"], run_info={})
+    logger.test_start("test-id-1")
+    time.sleep(0.125)
```

Comment:
I suppose you've tried to and couldn't mock out the timestamps? Sleeping 0.125 second is *probably* fine but definitely not my favourite.

Reply:
Given they're generated internally to `mozlog` I honestly didn't even try.